src/utils/transactions/TransactionResultUtil.ts

```diff
@@ -19,6 +20,7 @@ export enum OutcomeEnum {
   INVALID_METHOD = "12",
   KO_RETRIABLE = "13",
   INVALID_SESSION = "14",
+  TAKE_IN_CHARGE = "15",
```

Comment:
[bikeshedding]
What about
```suggestion
  TAKEN_IN_CHARGE = "15",
```
?

Reply:
That's reasonable